config/initializers/devise.rb

```diff
@@ -233,6 +233,10 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
+  config.omniauth :facebook, "385815088277667", "d1429b4b804fdb0c52c03128965440e8"
```

Comment:
should probably not have the secret in the file like this. you could set a heroku environment variable for it and then just use that here instead.
set the heroku variable with config:set APP_SECRET="whatever the secret is"
then in the initializer use   <%= ENV['APP_SECRET'] %>    where the secret is now